sql/moz-fx-data-shared-prod/mozilla_org_derived/gclid_conversions_v1/query.sql

```diff
@@ -0,0 +1,73 @@
+WITH gclids_to_ga_ids AS (
+  SELECT
+    gclid,
+    ga_client_id,
+    stub_session_id,
+  FROM
+    mozilla_org_derived.ga_sessions_v1
+  CROSS JOIN
+    UNNEST(all_reported_stub_session_ids) AS stub_session_id
+  WHERE
+    session_date >= DATE_SUB(@activity_date, INTERVAL @conversion_window DAY)
+    AND gclid IS NOT NULL
+),
+ga_ids_to_dl_token AS (
+  SELECT
+    ga_client_id,
+    stub_session_id,
+    dl_token,
+  FROM
+    stub_attribution_service_derived.dl_ga_triplets_v1
+  WHERE
+    ga_client_id IS NOT NULL
+    AND stub_session_id IS NOT NULL
```

Comment:
nit: for clarity, I'd probably move the joins up from the final `SELECT` to here instead of doing multiple lookups at the end.

Reply:
I'm going to ask you about this during our 1:1, I'm not 100% sure what you mean

Reply:
Looking it over, I'd prefer to keep as-is (even though it's not required here due to inner joins, I do prefer to keep the filtering logic separate from the joins).